dali/ft/filecopy.cpp

```diff
@@ -2655,22 +2650,21 @@ using namespace Azure::Storage;
 using namespace Azure::Storage::Files;
 using namespace Azure::Storage::Blobs;
 
-enum ApiCopyStatus { Pending, Success, Failed, Aborted };
+enum class ApiCopyStatus { Pending, Success, Failed, Aborted };
 interface IAPICopyClient : implements IInterface
 {
-    virtual ApiCopyStatus startCopy(const char *target, const char * source) = 0;
+    virtual void startCopy(const char *target, const char * source) = 0;
     virtual ApiCopyStatus getProgress(CDateTime & dateTime, int64_t & outputLength) = 0;
     virtual ApiCopyStatus abortCopy() = 0;
     virtual ApiCopyStatus getStatus() const = 0;
 };
 
-class CApiCopyClient : implements IAPICopyClient, public CInterface
+class CApiCopyClient : public CInterfaceOf<IAPICopyClient>
 {
 protected:
-    ApiCopyStatus status;
+    ApiCopyStatus status = ApiCopyStatus::Pending;
```

Comment:
This doesn't look right, because at construction the file copy hasn't started.  It should really have an extra state for not started.